js/gdm/authPrompt.js

```diff
@@ -600,18 +608,36 @@ const AuthPrompt = new Lang.Class({
         this.emit('cancelled');
     },
 
+    _loadCustomerSupportFile: function() {
+        if (this._customerSupportKeyFile)
+            return this._customerSupportKeyFile;
+
+        for (let path of CUSTOMER_SUPPORT_LOCATIONS) {
+            let keyFile = new GLib.KeyFile();
+            try {
+                keyFile.load_from_file(path, GLib.KeyFileFlags.NONE);
+                this._csutomerSupportKeyFile = keyFile;
+                return keyFile;
+            } catch (e) {
+                logError(e, 'Failed to read customer support data from ' + path);
+            }
+        }
+
+        return null;
+    },
+
     _ensureCustomerSupportData: function() {
         if (this._customerSupportEmail)
             return true;
 
-        const CUSTOMER_SUPPORT_FILENAME = 'vendor-customer-support.ini';
         const CUSTOMER_SUPPORT_GROUP_NAME = 'Customer Support';
         const CUSTOMER_SUPPORT_KEY_EMAIL = 'Email';
```

Comment:
I think you should move these other constants to the top as well, for consistency (and to have everything in one place)